### PR TITLE
httpflow 0.0.3 (new formula)

### PR DESCRIPTION
**Pull request by six-ddc**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/httpflow.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  homepage "https://github.com/six-ddc/httpflow"
+  url "https://github.com/six-ddc/httpflow/archive/0.0.3.tar.gz"
+  sha256 "26e8d1f8d6c0742b552bc333edd340dc3b3e3ad4590fe2e0e125d830142f0b37"
+  head "https://github.com/six-ddc/httpflow.git", :branch => "master"
+
+  def install
+    args = %W[
+      PREFIX=#{prefix}
+      CXX=#{ENV.cxx}
+    ]
```
Comment (bfontaine) [inline Formula/httpflow.rb]: Please inline the arguments instead of creating an array.

File: Formula/httpflow.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+    system "make"
+    system "make", "install", *args
+  end
+
+  test do
+    system "#{bin}/httpflow", "-h"
```
Comment (bfontaine) [inline Formula/httpflow.rb]: Can’t we write a test that does something more substantial than `-v`/`-h`?

File: Formula/httpflow.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
+class Httpflow < Formula
+  desc "Packet capture and analysis utility similar to tcpdump for HTTP"
+  homepage "https://github.com/six-ddc/httpflow"
+  url "https://github.com/six-ddc/httpflow/archive/0.0.3.tar.gz"
+  sha256 "26e8d1f8d6c0742b552bc333edd340dc3b3e3ad4590fe2e0e125d830142f0b37"
+  head "https://github.com/six-ddc/httpflow.git", :branch => "master"
```
Comment (bfontaine) [inline Formula/httpflow.rb]: Please remove `, :branch => "master"`, it’s the default.

**Review by bfontaine — CHANGES_REQUESTED** [review]

File: Formula/httpflow.rb (hunk 3 above)
Comment (six-ddc) [inline Formula/httpflow.rb]: OK, Thanks.

File: Formula/httpflow.rb (hunk 1 above)
Comment (six-ddc) [inline Formula/httpflow.rb]: OK, Thanks.

File: Formula/httpflow.rb (hunk 2 above)
Comment (six-ddc) [inline Formula/httpflow.rb]: It's not easy, because this require root permission to register network interface.

**Comment by six-ddc** [comment]
@bfontaine Thanks for the review, I updated it.

**Review by bfontaine — APPROVED** [review]

**Comment by fxcoudert** [comment]
Many thanks @six-ddc for your contribution to Homebrew!

**Pull request closed without merging**